README.rst

```diff
@@ -63,44 +63,48 @@ Usage
 
 .. code-block::
```

Comment:
we should change example usage photo.

Reply:
You're definitely right about that, I'll make a new one :)